frontend/ee/components/UsersPage/UsersTable.jsx

```diff
@@ -0,0 +1,177 @@
+                        }}
+                        data-cy="user-name"
+                      >
+                        {user.name}
+                      </span>
+                    </td>
+                    <td className="text-muted">
+                      <a className="text-reset user-email" data-cy="user-email">
+                        {user.email}
+                      </a>
+                    </td>
+                    {user.status && (
+                      <td className="text-muted">
+                        <span
+                          className={`badge bg-${
+                            user.status === 'invited' ? 'warning' : user.status === 'archived' ? 'danger' : 'success'
+                          } me-1 m-1`}
+                          data-cy="status-badge"
+                        ></span>
+                        <small className="user-status" data-cy="user-status">
+                          {user.status}
+                        </small>
+                        {user.status === 'invited' && 'invitation_token' in user ? (
+                          <CopyToClipboard text={generateInvitationURL(user)} onCopy={invitationLinkCopyHandler}>
+                            <img
+                              data-tip="Copy invitation link"
+                              className="svg-icon"
+                              src="assets/images/icons/copy.svg"
+                              width="15"
+                              height="15"
+                              style={{
+                                cursor: 'pointer',
+                              }}
+                              data-cy="copy-invitation-link"
+                            ></img>
+                          </CopyToClipboard>
+                        ) : (
+                          ''
+                        )}
+                      </td>
+                    )}
+                    <td>
+                      <button
+                        type="button"
+                        style={{ minWidth: '100px' }}
+                        className={`btn btn-sm btn-outline-${user.status === 'archived' ? 'success' : 'danger'} ${
+                          unarchivingUser === user.id || archivingUser === user.id ? 'btn-loading' : ''
+                        }`}
+                        disabled={unarchivingUser === user.id || archivingUser === user.id}
+                        onClick={() => {
+                          user.status === 'archived' ? unarchiveOrgUser(user.id) : archiveOrgUser(user.id);
+                        }}
+                        data-cy="user-state"
+                      >
+                        {user.status === 'archived' ? 'Unarchive' : 'Archive'}
+                      </button>
+                    </td>
+                  </tr>
+                ))}
+              </tbody>
+            )}
+          </table>
+        </div>
+      </div>
+      <div className="users-pagination">
+        {meta.total_count > 10 && (
```

Comment:
I can see you have use `decamelizeKeys` in the controller? should it be `meta.totalCount` instead?

Reply:
no,i am gettings the data in front-end like this